publication-event-dtopublicationdoi-producer/src/test/resources/dynamodbevent_new_only.json

```diff
@@ -514,6 +514,49 @@
           "ss": null,
           "ns": null,
           "bs": null
+        },
+        "doiRequest": {
+          "s": null,
+          "n": null,
+          "b": null,
+          "m": {
+            "status": {
+              "s": "REQUESTED",
```

Comment:
Example of enum with CAPITAL letters

Reply:
Yes, we have different practice for PublicationStatus and DoiRequestStatus right now. We need to make a decision.

Reply:
updated to use CAPTIAL letters for both enums, but this change is not reflected in the datamodel yet.